renderer/src/typings.ts

```diff
@@ -1,13 +1,14 @@
-import { Activity, FILTransaction, FILTransactionProcessing } from '../../shared/typings'
+import { FILTransaction, FILTransactionProcessing } from '../../shared/typings'
+import { ActivityEvent } from '@filecoin-station/core'
```

Comment:
Yes! Unfortunately there was a naming conflict in `core`: `Activity` is now the class for interfacing with activities, `ActivityEvent` is one entry. I guess we can rename the type for us here? Or would you propose a different solution?

Reply:
Yes! Unfortunately there was a naming conflict in `core`: `Activity` is now the class for interfacing with activities, `ActivityEvent` is one entry. I guess we can rename the type for us here? Or would you propose a different solution?

Reply:
Ah, that naming conflict is annoying. Maybe the class for interfacing with activities should be called differently, perhaps `ActivityStore` or `ActivityClient`.

Would it be too confusing to rename the types on our (Desktop) side only?

- At minimum, we can rename `ActivityEvent` to `Activity` in renderer. I am assuming we are not accessing the Core Activity class directly from the rendered, right? Would it be confusing if rendered was using Activity for something else than the main process?
- We can also rename `ActivityEvent` to `Activity` and `Activity` to `ActivityClient` (or `ActivityStore`) everywhere in the Desktop codebase. I think this would be the cleanest option.
- Or we can keep what you have right now, it's not that bad.

Reply:
I've renamed the type. I think it works quite well because we're using the shared type file, where there's only one source of truth.

In the future we can rename the `core` type to `ActivityStore`, I like that 👍